go/Dockerfile

```diff
@@ -1,4 +1,3 @@
+FROM fnproject/go:dev as build-stage
 FROM fnproject/base:edge
-
-# The Go image doesn't need anything else.
-# Use fnproject/go:dev to build.
+COPY --from=build-stage /usr/local/go/lib/time/zoneinfo.zip /usr/local/go/lib/time/zoneinfo.zip
```

Comment:
see 2 issues here, we're changing the base layer here to `fnproject/go:dev` which is different that `fnproject/base:edge` and may have consequences.

also seems like the easy thing to do here is update `fnproject/base:edge` to install the `tzdata` package (just `apk-add tzdata`) instead of getting this only for go specifically. presumably, other languages are going to have very similar code to get the time zone data from the system (ZONEINFO env var, etc) and this only works for go, when the wider unix community generally looks in `/usr/share/zoneinfo` or similar.

Reply:
we've done similar for things like certs, in the past (add to base image)